search/serializers.py

```diff
@@ -676,3 +704,26 @@ def serialize_user_list_for_bulk(user_list_obj):
         "_id": gen_user_list_id(user_list_obj),
         **ESUserListSerializer(user_list_obj).data,
     }
+
+
+def serialize_bulk_videos(ids):
```

Comment:
Maybe add a test for this and `serialize_video_for_bulk`, similar to  `test_serialize_bulk_courses`, `test_serialize_course_for_bulk`, etc